api/CacheStorage.json

```diff
@@ -321,6 +321,51 @@
             "deprecated": false
           }
         }
+      },
+      "secure_context_required": {
+        "__compat": {
+          "description": "Secure context required",
+          "support": {
+            "chrome": {
+              "version_added": true
```

Comment:
This landed in 65: https://storage.googleapis.com/chromium-find-releases-static/c32.html#c3231dc7d9d57c5dcf5dc0f0a6bb4af48aba85a4